projects/demo/src/pages/stackblitz/files/styles.css

```diff
@@ -1,7 +1,43 @@
-/* TODO: make better basic customization */
+* {
```

Comment:
It will duplicate this CSS-variables for every tag.

Should it be `:root {` or just `body` ?